### PR TITLE
docs: add backstage documentation

### PR DESCRIPTION
**Pull request by svdsande**

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: "backstage.io/v1alpha1"
+kind: "Component"
+metadata:
+  name: "ember-redux"
+  description: |
+    Ember bindings for Redux
+
+    During our Ember 4.0 upgrade we ran into some issues with ember-redux, and we had to fork it in order to fix it.
+    These fixes are also available in the upstream repo, but they are not released yet.
+    See this pull request for further details: https://github.com/ember-redux/ember-redux/pull/213
```
Comment (Frank3K) [inline catalog-info.yaml]: There's very little space in backstage for displaying the description. See if you (or AI) can make it shorter.

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  annotations:
+    "backstage.io/source-location": "url:https://github.com/gynzy/ember-redux"
+    "backstage.io/edit-url": "https://github.com/gynzy/ember-redux/tree/master/catalog-info.yaml"
+    "backstage.io/view-url": "https://github.com/gynzy/ember-redux/tree/master/catalog-info.yaml"
+    tags: [frontend, "ember", "redux"]
+spec:
+  type: "library"
+  lifecycle: "production"
+  owner: "guild-frontend"
```
Comment (Frank3K) [inline catalog-info.yaml]: Do the values need to be quoted?

**Review by Frank3K — CHANGES_REQUESTED** [review]
This yaml does not pass the analyzer.

image

File: catalog-info.yaml (hunk 2 above)
Comment (svdsande) [reply catalog-info.yaml]: Stripped all quotes to make sure it is consistent

File: catalog-info.yaml (hunk 1 above)
Comment (svdsande) [reply catalog-info.yaml]: Done. Let me know what you think 👍

**Review by Frank3K — APPROVED** [review]
LGTM and valid ✅

**Pull request merged**